### ISSUE TITLE
Config parser crashes

### ISSUE THREAD

**Issue by br-olf** [firecat53/keepmenu#56]
Hello I encountered the following error and can not start keepmenu any longer.
It occurred after my home partition ran out of space but my config is unaltered under .config/keepmenu/config.ini
I freed some space and tried reinstalling keepmenu. Also a system reboot didn't help.

```
% keepmenu                      
Traceback (most recent call last):
  File "/usr/lib/python3.8/configparser.py", line 789, in get
    value = d[option]
  File "/usr/lib/python3.8/collections/__init__.py", line 898, in __getitem__
    return self.__missing__(key)            # support subclasses that define __missing__
  File "/usr/lib/python3.8/collections/__init__.py", line 890, in __missing__
    raise KeyError(key)
KeyError: 'port'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/keepmenu", line 1421, in <module>
    MANAGER = client()
  File "/usr/bin/keepmenu", line 1231, in client
    port, auth = get_auth()
  File "/usr/bin/keepmenu", line 174, in get_auth
    port = int(auth.get('DEFAULT', 'port'))
  File "/usr/lib/python3.8/configparser.py", line 792, in get
    raise NoOptionError(option, section)
configparser.NoOptionError: No option 'port' in section: 'DEFAULT'
```

**Comment by br-olf** [comment]
I found the issue:
Because of the full home directory the auth file `.cache/.keepmenu-auth` got corrupted.
Removing this file fixed the issue.

**Comment by firecat53** [comment]
Closed with #57
